university_events.yml

```diff
@@ -0,0 +1,5 @@
+# Store university events here. All fields are required.
+- name: Example
+  location: Imaginary place
+  start_date: 2017-01-20
+  end_date: 2017-01-22
```

Comment:
Are there any actual events that we need to populate in this file prior to merge?